sourcemash/api/items.py

```diff
@@ -32,23 +34,20 @@ def output(self, key, item):
 item_fields = {
     'id': fields.Integer,
     'title': fields.String,
-    'text': fields.String,
     'link': fields.String,
     'last_updated': fields.DateTime,
     'author': fields.String,
     'category_1': fields.String,
     'category_2': fields.String,
-    'summary': fields.String,
+    'voteSum': fields.Integer,
     'image_url': fields.String,
+    'summary': fields.String,
     'feed': fields.Nested(feed_fields),
     'unread': getUnreadStatus,
     'saved': getSavedStatus,
-    'vote': getVote,
-    'voteSum': fields.Integer,
-    'uri': fields.Url('api.item')
```

Comment:
Correct. And we're not ever passing the text to the frontend (nor should we be).